bin/publish-swagger-docs.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Runs the api and backend db, grabs the generated swagger json spec and compares to what is in the
+# central reform-api-docs repo. Updates reform-api-docs spec if needed
+
+set -e
+
+REPO_NAME=$(echo ${TRAVIS_REPO_SLUG} | cut -f2- -d/)
+
+if [ "$REPO_NAME" == "spring-boot-template" ]; then
+    exit 0;
+fi
```

Comment:
would be good to add a todo comment here for users to remove this block imho

Reply:
same as port, really. tbh it would be great to not have to amend this script